longest_word/game.py

```diff
@@ -19,4 +20,12 @@ def is_valid(self, word: str) -> bool:
         """Check if a given world is valid"""
         if not isinstance(word, str):
             return False
-        return all(x in self.grid for x in list(word))
+        # check if the given  world is a subset of the desired solution
+        if not all(x in self.grid for x in list(word)):
```

Comment:
do we need to cast into a list ? ;)

Reply:
does this handle individual letter count ? ( "GROOVE" invalid for grid "GROVEABC" )

Reply:
changed the implementation to match proposed solution for this :p